CLASSROOM SCHEDULER WEBSITE/README.md

```diff
@@ -0,0 +1,27 @@
+<h1>Classroom Website 👩‍🏫</h1>
+
+<p>A website that lets you write about all your classes at one place. You can also search by typing the name of the teacher if you have so many classes to take.</p>
+
+<h3>Used Technologies</h3>
+<ul>
+  <li>HTML5</li>
+  <li>CSS3</li>
+  <li>JavaScript</li>
+</ul>
+
```

Comment:
Add a "Use of this project" section over here, that describes places this project can be used.